spec/unit/puppet/provider/selinux_state/selinux_state_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'spec_helper'
+
+describe Puppet::Type.type(:selinux_state).provider(:selinux_state) do
+
+  let(:provider) { resource.provider }
+  let(:resource) {
+    Puppet::Type.type(:selinux_state).new(
+      name: 'set_selinux_state',
+      autorelabel: true
+    )
+  }
+
+  describe 'relabel?' do
+    context 'enforcing -> enforcing' do
+      it 'does not need relabeling' do
+        provider.stubs(:ensure).returns 'enforcing'
+        expect(provider.relabel?('enforcing')).to be_falsey
+      end
+    end
+
+    context 'enforcing -> permissive' do
+      it 'does not need relabeling' do
+        provider.stubs(:ensure).returns 'enforcing'
+        expect(provider.relabel?('permissive')).to be_falsey
+      end
+    end
+
+    context 'enforcing -> disabled' do
+      it 'does not need relabeling' do
+        provider.stubs(:ensure).returns 'enforcing'
+        expect(provider.relabel?('disabled')).to be_falsey
+      end
+    end
+
+    context 'permissive -> enforcing' do
+      it 'needs relabeling' do
+        provider.stubs(:ensure).returns 'permissive'
+        expect(provider.relabel?('enforcing')).to be_truthy
+      end
+    end
+
+    context 'permissive -> permissive' do
+      it 'does not need relabeling' do
+        provider.stubs(:ensure).returns 'permissive'
+        expect(provider.relabel?('permissive')).to be_falsey
+      end
+    end
+
+    context 'permissive -> disabled' do
+      it 'does not need relabeling' do
+        provider.stubs(:ensure).returns 'permissive'
+        expect(provider.relabel?('disabled')).to be_falsey
+      end
+    end
+
+    context 'disabled -> enforcing'do
```

Comment:
[I can see it passing in Travis](https://travis-ci.org/simp/pupmod-simp-selinux/jobs/280078050#L1527-L1528), but I'm a bit surprised this works without any white space between the `'` and the `do`

Reply:
😃 I'll fix it